docs/nightly_releases.md

```diff
@@ -67,7 +67,7 @@ python3.11 -m venv 3.11.venv
 source 3.11.venv/bin/activate
 
 # Install 'sharktank' package from nightly releases.
-python -m pip install sharktank -f https://github.com/nod-ai/SHARK-Platform/releases/expanded_assets/dev-wheels
+pip install sharktank -f https://github.com/nod-ai/SHARK-Platform/releases/expanded_assets/dev-wheels
```

Comment:
I've heard tell of a shark-ai package though don't see it yet in the nightly wheels. Not blocking but wondering when that'll be there.

Reply:
https://pypi.org/project/shark-ai/

discussion about publishing to nightly: https://github.com/nod-ai/SHARK-Platform/pull/475#discussion_r1837044406

(meta package is a bit special when it comes to packaging... not sure if/how we want to publish it nightly too)

Reply:
ACK I just want whatever instructions we have here to work with the SDXL workflow. Right now if we follow this guide and the SDXL guide iree-compiler, fastapi, and uviron aren't installed. So we should either have the meta package here or ensure these deps get pulled in / ask folks to install them as prereqs when starting with the nightlies.

Reply:
Yep. I'll keep iterating on this over the next few days but wanted to checkpoint progress. There is still no mention of just `pip install shortfin` (or any of the other stable packages).

Reply:
If we want to publish nightly releases of the shark-ai package, that's easily doable. Regarding the dependencies required for the SDXL workflow, I assume we might want to add so-call extra requires to shortfin (`pip install shortfin[{app.sdxll,sdxl,sd}]`) that pulls in directly used deps like `fastapi`. The same options can be added to the shark-ai package. Other deps like `iree-compiler`get pulled in by shark-ai already by default.